packages/transformers/js/core/src/env_replacer.rs

```diff
@@ -126,7 +126,6 @@ impl<'a> Fold for EnvReplacer<'a> {
           }
         }
         PatOrExpr::Expr(expr) => Some(&**expr),
-        _ => None,
```

Comment:
(This was a preexisting warning because the `PatOrExpr` enum only has two members and the switch is therefore already exhaustive)